Importers/Rhino/Speckle.Importers.Rhino/Internal/FileTypeConfig/Rhino3dmConfig.cs

```diff
@@ -0,0 +1,23 @@
+using Rhino;
+using Speckle.Sdk;
+
+namespace Speckle.Importers.Rhino.Internal.FileTypeConfig;
+
+/// <summary>
+/// <see cref="RhinoDoc.OpenHeadless(string)"/> will preserve the units defined by the file
+/// </summary>
+public sealed class Rhino3dmConfig : IFileTypeConfig
+{
+  public RhinoDoc OpenInHeadlessDocument(string filePath)
+  {
+    RhinoDoc? doc = RhinoDoc.OpenHeadless(filePath);
```

Comment:
Using `OpenHeadless` rather than `CreateHeadless` + `Import` preserves units for 3dm files